driver/src/price_estimation.rs

```diff
@@ -261,10 +270,13 @@ mod tests {
             .expect_get_prices()
             .withf(|tokens| tokens == [Token::new(2, "USDT", 6)])
             .returning(|_| {
-                Ok(hash_map! {
-                    TokenId(1) => 1_000_000_000_000_000_000,
-                    TokenId(2) => 1_000_000_000_000_000_000,
-                })
+                async {
```

Comment:
nit: Given how often we write `async { ... }.boxed()` in the tests, maybe worth while creating a macro for this?

Reply:
We could put a `async_boxed!` macro in utils maybe but not in this PR.